core/src/main/java/com/es/core/service/phone/PhoneService.java

```diff
@@ -3,12 +3,16 @@
 import com.es.core.model.phone.Phone;
 import com.es.core.dao.phone.SortField;
 import com.es.core.dao.phone.SortType;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
 
 
 import java.util.List;
 
 public interface PhoneService {
-    List<Phone> findAll(int offset, int limit);
+    Phone get(Long id);
 
-    List<Phone> sort(SortField sortField, SortType sortType);
+    List<Phone> findAll(int offset, String userSearch, SortField sortField, SortType sortType);
```

Comment:
Not sure you need both `List<Phone> findAll()` and `Page<Phone> findAll()`, pick one. Probably page related is the best choice.